README.md

```diff
@@ -98,6 +98,23 @@ If you want to run Clippy **only** on the given crate, use the `--no-deps` optio
 cargo clippy -p example -- --no-deps 
 ```
 
+### As a rustc wrapper (`clippy-driver rustc`)
+
+Clippy can also be used in projects that do not use cargo. To do so, you will need to wrap
+your `rustc` compilation commands with `clippy-driver`. For example, if your project runs:
+
+```terminal
+rustc --edition 2018 -Cpanic=abort foo.rs
+```
+
+Then, to enable Clippy, you will need to call:
+
+```terminal
+clippy-driver rustc --edition 2018 -Cpanic=abort foo.rs
```

Comment:
```suggestion
clippy-driver --edition 2018 -Cpanic=abort foo.rs
```
also works. This does only work with `rustc` as a second argument, because the clippy-driver has a hack that removes this.

Reply:
I noticed that, but I didn't know it was the preferred way -- that is good to know! I will add a note for that also.

Reply:
Changed the title of the subsection to match, too.

Reply:
I wouldn't note that, because it is an implementation detail necessary to make clippy work.

The `clippy-driver` binary is the same as the `rustc` binary from the user/CLI perspective, just that it runs more lints.

Reply:
Done!

What do you mean by necessary? From what I understand, the hack around the second argument isn't be necessary but it is there for `cargo clippy`, no?

Anyway, I agree the chances of someone hitting that behavior by mistake without intending to do so are very low; however, it is still user-facing behavior. For me, as a user, it is OK if it is not supported but I'd prefer to know about it (and users may still notice via cargo verbose output anyway, like I did, and wonder what is going on).

Reply:
It is necessary for `cargo clippy` to work, yes. It's because `cargo` for some reason runs `clippy-driver rustc`, so Clippy has to get rid of the `rustc`.

It doesn't hurt that adding `rustc` to the command is possible. But I don't see a reason, why we should advertise it. 

> (and users may still notice via cargo verbose output anyway, like I did, and wonder what is going on)

If you're interested in that, you could ask why it is implemented like this on Zulip or on other channels. IMO it's the same question as why cargo adds `-C extra-filename=-73486377a7462660` to the command.